stacker/session_cache.py

```diff
@@ -23,18 +32,16 @@ class CredentialCache(object):
     """
 
     CACHE_DIR = os.path.expanduser(
-        os.path.join('~', '.stacker', 'cache'))
+        os.path.join('~', '.aws', 'cli', 'cache'))
```

Comment:
Is it worth putting some debug logging in here so that people can debug in case things start breaking?